### PR TITLE
Add passenger ruby gem (CVE-2015-7519)

### PR DESCRIPTION
**Pull request by skorth**
It has been a long time since my last commit. Please double check.

### CODE REVIEW

File: gems/passenger/CVE-2015-7519.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+gem: passenger
+cve: 2015-7519
+url: https://blog.phusion.nl/2015/12/07/cve-2015-7519/
+title: Phusion Passenger Server allows to overwrite headers in some cases
+date: 2015-11-23
+description: It was discovered by the SUSE security team that it was possible,
```
Comment (reedloden) [inline gems/passenger/CVE-2015-7519.yml]: Not sure mentioning the SUSE team here is needed.

Comment (skorth) [inline gems/passenger/CVE-2015-7519.yml]: @reedloden I removed it and added some extra text. Hope thats ok.

**Comment by phillmv** [comment]
Eh, if attribution is upstream/legit I'm OK with leaving the reference to the SUSE security team.

**Comment by reedloden** [comment]
I like the new description much better. We can add SUSE back later if needed. Merged!

**Pull request merged**